packages/core/package.json

```diff
@@ -115,7 +115,6 @@
     "framer-motion": "^6.5.1",
     "jest-styled-components": "^7.1.1",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.8.8",
```

Comment:
This is needed afaik for vscode to function correctly

Reply:
Are you setting this to anything in your VSCode settings? I just realized I was getting results inconsistent with our Prettier config when pointing it to `./common/autoinstallers/rush-prettier/node_modules/prettier`. I'll leave this dependency as is just to be safe.